pages/tidal tools/getstarted.md

```diff
@@ -77,9 +77,13 @@ methods below.
 
 #### Configuration file
 
+
 Alternatively, you can use the `tidal config` command to manually set your
 credentials.
 
+It is  Configuration file is not encrypted.
```

Comment:
grammars

Reply:
That line is no longer needed. Thanks for picking that up. 
```suggestion
```